dapr/proto/runtime/v1/dapr.proto

```diff
@@ -216,6 +219,22 @@ message GetSecretResponse {
   map<string, string> data = 1;
 }
 
+// GetBulkSecretRequest is the message to get the secrets from secret store.
+message GetBulkSecretRequest {
+  // The name of secret store.
+  string store_name = 1;
+
+  // The metadata which will be sent to secret store components.
+  map<string,string> metadata = 2;

```

Comment:
Shouldn't this Request envelope hold an array of keys? I don't see where the array of keys are specified.

Reply:
The idea of the new API was to get all the secrets of a given store (https://github.com/dapr/components-contrib/pull/310). 

Many of my customers use a dedicated store per app and they are used to get all secrets at once. For instance those who are using the Key Vault configuration provider for .NET expect it to load all secrets and merge it with the apps configuration and that is something I'm looking forward to add to this PR:   https://github.com/dapr/dotnet-sdk/pull/465 as discussed here: https://github.com/dapr/dapr/issues/2168.